test/parallel/test-urlpattern.js

```diff
@@ -26,3 +26,7 @@ throws(() => {
 }, {
   message: 'boom'
 });
+
+throws(() => {
+  new URLPattern('invalid value');
+}, /TypeError: Failed to construct URLPattern/);
```

Comment:
I prefer if we didn't add a test for an exact error message, since it is not bounded by semantic versioning. We try to avoid having error message checks in tests in general, unless we have to distinguish 2 different errors from each other.
```suggestion
```

Reply:
Fair point 👍 since `TypeError` has already been tested in the `wpt` so here is only testing the error message. I am happy to remove it.